docs/reference/cluster/get-settings.asciidoc

```diff
@@ -37,7 +37,7 @@ defined, but can also include the default settings by calling the
 include::{es-ref-dir}/rest-api/common-parms.asciidoc[tag=flat-settings]
 
 `include_defaults`::
-    (Optional, Boolean) If `true`, returns default cluster settings from the local node. 
+    (Optional, Boolean) If `true`, returns default cluster settings from the local node.
```

Comment:
Sorry, IDE introduced noise. Will try and make that go away.

Reply:
This was due to `.editorconfig`, I temporarily disabled it for this PR but it will happen.